src/runtime_src/tools/xclbinutil/XclBinSignature.cxx

```diff
@@ -493,6 +498,9 @@ void verifyXclBinImage(const std::string& _fileOnDisk,
   sk_X509_push(ca_stack, x509);
 
   if (!PKCS7_verify(p7, ca_stack, store, bmImage, NULL, PKCS7_DETACHED |  PKCS7_BINARY | PKCS7_NOINTERN)) {
+    long err = ERR_peek_last_error();
```

Comment:
Thank you for adding this.  It will definitely assist the users, WTS, and SW Apps in the future to resolve failing signatures.